### PR TITLE
Release Notes 1.45.5

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -45,6 +45,46 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- For Winrm deployment with Command step and auth type as Kerberos if the environment variables contains characters `\b, \v, \c, \f, &` the script execution was failing as we parse the response of script execution to xml string and above characters are illegal xml characters. The issue is fixed and now and these characters will be escaped. This change is behind the FF `CDS_ESCAPE_ENV_VARS_FOR_WINRM_KERBEROS_NG`. (CDS-97690, ZD-55276,58201,66326)
+- While using a connector in Project scope during pre-flight check, if the check fails connector reference hyperlinks to Connector URL at account level due to which it routes to a Not Found Page. This was happening because scope details were missing from the api. The issue is fixed now. (CDS-97593, ZD-64673)
+- Logs were not being uploaded for shell script and some other steps in case the step timed out. The issue is fixed and shell script now uploads the logs on step timeout. (CDS-97521, ZD-64422)
+- Shell script step was not doing a capability check before assigning the tasks to the delegate. The issue is fixed and shell script step will now have host capibility check. (CDS-97512, ZD-66326,66349)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:84]:
```suggestion
- The shell script step was not doing a capability check before assigning the tasks to the delegate. The issue is fixed and the shell script step will now have a host capability check. (CDS-97512, ZD-66326,66349)
```

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -45,6 +45,46 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- ServiceNow step was not updating the ticket yet was showing successful and no logs was being shown. The issue is fixed now and console logs and debugging help has been added in the ServiceNow steps. (CDS-97033, ZD-63637)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:85]:
```suggestion
- The ServiceNow step was not updating the ticket but was showing successful with no logs being shown. The issue is now fixed and console logs and debugging help has been added in the ServiceNow steps. (CDS-97033, ZD-63637)
```

**Review by SushrutHarness — APPROVED** [review]
Left suggestions, but approving to not be a release blocker

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -45,6 +45,50 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
 
+## July 2024
+
+### Version 1.45.5
+
+#### New Features and Enhancement
+
+**Bi-Directional Sync Health Status for Git Sync**
+
+We have introduced a Git Sync status page for Harness NG. Once you have synced your Harness entities with your Git repository, you can view bidirectional activity between Harness and your repository using the Bi-Directional Sync Health Status feature. For each entitiy, you can see file path, commit message, status of the last sync as well as the payload. Refer to Harness [Git Sync health Page](/docs/platform/git-experience/git-sync-health-page.md) for more information. (PIPE-16865)
```
Comment (SushrutHarness) [inline release-notes/continuous-delivery.md:56]:
```suggestion
We have introduced a Git Sync status page for Harness NG. Once you have synced your Harness entities with your Git repository, you can view bidirectional activity between Harness and your repository using the Bi-Directional Sync Health Status feature. For each entitiy, you can see file path, commit message, status of the last sync as well as the payload. Refer to Harness [Git Sync health Page](/docs/platform/git-experience/git-sync-health-page) for more information. (PIPE-16865)
```

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -45,6 +45,55 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 </details>
 
+## July 2024
+
+### Version 1.45.5
+
+#### New Features and Enhancement
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:52]:
```suggestion
#### New features and enhancements
```

File: release-notes/continuous-delivery.md [hunk 5]
```diff
@@ -45,6 +45,55 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+**Bi-Directional Sync Health Status for Git Sync**
+
+We have introduced a Git Sync status page for Harness NG. Once you have synced your Harness entities with your Git repository, you can view bidirectional activity between Harness and your repository using the Bi-Directional Sync Health Status feature. For each entitiy, you can see file path, commit message, status of the last sync as well as the payload. Refer to Harness [Git Sync health Page](/docs/platform/git-experience/git-sync-health-page) for more information. (PIPE-16865)
+
+
+**HTTP support HTTPS Authentication for Terraform and Terragrunt Modules**
+
+You can now fetch modules for Terraform and Terragrunt via HTTPS authentication using GitHub, Bitbucket, and GitLab connectors(CDS-95361)
+
+#### Behavior Changes
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:63]:
```suggestion
#### Behavior changes
```

File: release-notes/continuous-delivery.md [hunk 6]
```diff
@@ -45,6 +45,55 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- Previously, when the verify step failed, and an action was taken based on the failure strategy or manual intervention, the title was always shown as **Manual Intervention**. (CDS-97985, ZD-65113)
+
+   Now, the title is updated to reflect the nature of the intervention:
+   - **Intervention** is displayed when the action is performed through the CV failure strategy configuration.
+
+   - **Manual Intervention** is displayed when the action is performed through manual intervention.
+
+#### Fixed Issue
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:72]:
```suggestion
#### Fixed issues
```

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Pull request merged**